src/e4stypes/order_information.py

```diff
@@ -2,20 +2,27 @@
 from typing import List
 from .database import Database
 from .item import Item
+from bson import ObjectId
 
 
 class OrderInformation:
     def __init__(self, item_list: List[Item], total_amount: Decimal):
         self.item_list: List[Item] = item_list
         self.total_amount: Decimal = total_amount
+        self.buyer = None
 
     def add_to_cart(self, item_id: str) -> type(None):
-        self.item_list.append(item_id)
-        return self.item_list
+        item = Database.get_item_by_id(item_id)
+        item_list_ids = []
+        for item1 in self.item_list:
+            item_list_ids.append(item1.get_item_id())
+        if ObjectId(item_id) not in item_list_ids:
+            self.item_list.append(item)
```

Comment:
```suggestion
        item_to_add = Database.get_item_by_id(item_id)
        item_list_ids = []
        for item in self.item_list:
            item_list_ids.append(item.get_item_id())
        if ObjectId(item_id) not in item_list_ids:
            self.item_list.append(item_to_add)
```

lets try avoiding variable names like `item1`. since there's a semantic difference between the two items, how about `item_to_add` for the item that will be added?